std/xml.d

```diff
@@ -2858,6 +2862,14 @@ private
         s = s[1..$];
     }
 
+    char reqs(ref string s, ref string chars)
```

Comment:
I'd expect `reqs` to mean "require string", but it's more like "require one of these characters". How about `requireOneOf`, or just overload `reqc`. Also, the `chars` parameter shouldn't be `ref`.